ci/terraform/oidc/authentication_callback.tf

```diff
@@ -0,0 +1,69 @@
+module "frontend_api_authentication_callback_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "frontend-api-authentication-callback-role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = [
+    aws_iam_policy.audit_signing_key_lambda_kms_signing_policy.arn,
+    aws_iam_policy.dynamo_user_read_access_policy.arn,
+    aws_iam_policy.dynamo_user_write_access_policy.arn,
+    aws_iam_policy.dynamo_client_registry_read_access_policy.arn,
+    module.oidc_txma_audit.access_policy_arn
+  ]
+}
+
+module "authentication_callback" {
+  source = "../modules/endpoint-module"
+
+  endpoint_name   = "orchestration-redirect"
+  path_part       = "orchestration-redirect"
+  endpoint_method = "POST"
```

Comment:
Is this for sure? The lambda code seems to be assuming query parameters for e.g. code being passed, hence my personal assumption of `GET`